### ISSUE TITLE
opensuse: curl 60 error when refreshing repositories via zypper

### ISSUE THREAD

**Issue by thisch** [systemd/mkosi#133]
In order to create an opensuse image (`sudo mkosi -d opensuse -r 42.2 -t directory -o quux`), I had to apply the following patch to mkosi

```diff
--- a/mkosi
+++ b/mkosi
@@ -1091,7 +1091,7 @@ def install_opensuse(args, workspace, run_build_script):
     #
     # Install the "minimal" package set.
     #
-    subprocess.run(cmdline + ["-t", "pattern", "minimal_base"], check=True)
+    subprocess.run(cmdline + ["patterns-openSUSE-minimal_base"], check=True)
 
     #
     # Now install the additional packages if necessary.
```
But then  I get the following curl error if I call `zypper ref`

```
Abort, retry, ignore? [a/r/i/...? shows all options] (a): ^Cquux:/etc/zypp/repos.d # zypper ref
Download (curl) error for 'https://download.opensuse.org/distribution/leap/42.2/repo/oss/media.1/media':
Error code: Curl error 60
Error message: SSL certificate problem: unable to get local issuer certificate
```

**Comment by thisch** [comment]
Here is the output of the zypper call the lead to my mkosi patch
```
sudo zypper --root /home/thomas.hisch/.mkosi-jkm4yb6p/root --gpg-auto-import-keys install --no-recommends -t pattern minimal_base 

Automatically importing the following key:
Key ID: B88B2FD43DBDC284
Key Name: openSUSE Project Signing Key <opensuse@opensuse.org>
Key Fingerprint: 22C07BA534178CD02EFE22AAB88B2FD43DBDC284
Key Created: Mon 05 May 2014 10:37:40 AM CEST
Key Expires: Thu 02 May 2024 10:37:40 AM CEST
Repository: Main

Building repository 'Main' cache .....................................................................................................................[done]
Building repository 'Updates' cache ..................................................................................................................[done]
Loading repository data...
Warning: Repository 'Updates' appears to be outdated. Consider using a different mirror or server.
Reading installed packages...
'minimal_base' not found in package names. Trying capabilities.
No provider of 'pattern:minimal_base' found.
Resolving package dependencies...
```

**Comment by poettering** [comment]
@fbuihuu any idea about this one? appears to be an issue in the opensuse hookup?

**Comment by fbuihuu** [comment]
Isn't this the same issue as the one spotter here: https://github.com/systemd/mkosi/pull/109#pullrequestreview-48269494 ?

Running `mkosi -d opensuse -r 42.2 -t directory -o rootfs` works fine on TW.

**Comment by juaningan** [comment]
I also need the patch for both leap and tumbleweed

**Comment by johannbg** [comment]
@juaningan is this still an issue?

**Comment by DaanDeMeyer** [comment]
Fixed by #345
